modules/cassandra/src/main/scala/sup/cassandra.scala

```diff
@@ -0,0 +1,18 @@
+package sup
+
+import cats.Id
+import cats.effect.implicits.catsEffectSyntaxConcurrent
+import cats.effect.{Concurrent, Sync, Timer}
+import com.datastax.oss.driver.api.core.CqlSession
+
+import scala.concurrent.duration.FiniteDuration
+
+object cassandra {
+
+  def connectionCheck[F[_] : Sync : Concurrent](session: CqlSession, timeout: FiniteDuration)(implicit timer: Timer[F]): HealthCheck[F, Id] =
```

Comment:
please run scalafmt here as well (I recommend just enabling "format on save" in your editor :) )

Reply:
```suggestion
  def connectionCheck[F[_] : Concurrent: Timer](session: CqlSession, timeout: FiniteDuration): HealthCheck[F, Id] =
```

Reply:
i actually removed the timer altogether